CONTRIBUTING.md

```diff
@@ -31,7 +31,6 @@ In case you are suggesting a new feature, we will match your idea with our curre
   *Note: Node 6 or greater would be better for "best results".*
 * Fork the **webpack-cli** repo at [https://github.com/webpack/webpack-cli](https://github.com/webpack/webpack-cli).
 * `git clone <your-clone-url> && cd webpack-cli`
-* Install the commit validator: `npm run install-commit-validator`
 
 ### Setup with npm
 * Install the dependencies: `npm install`
```

Comment:
add `npm prepare` after the install statement, same for yarn

Reply:
ah - seems like prepare is a lifecycle hook for both yarn and npm I wasn't aware of.

https://github.com/yarnpkg/yarn/pull/3004
https://iamakulov.com/notes/npm-4-prepublish/

Reply:
gotcha! I tend to do it after installing based on habit 😆